eo-parser/src/test/resources/org/eolang/parser/packs/syntax/text-block.yaml

```diff
@@ -0,0 +1,38 @@
+xsls: []
+tests:
+  - //o[@base='string' and not(text())]
+  - //o[@base='string' and text()='simple']
+  - //o[@base='string' and normalize-space(text())='Hi, there Adiós']
+  - //o[@base='string' and normalize-space(text())='first second']
+  - //o[@base='string' and text()='   third']
+  - //o[@base='string' and text()='forth']
+eo: |
+  # Empty
+  """""" > txt
+
+  # Single line
+  """simple""" > txt
+
+  # Two lines
+  """
+  Hi, there
+  Adiós
+  """ > txt
+
+  # Preserve margin
+  stdout
+    """
+    first
+     second
+    """
+
+  # Preserve margin
+  stdout
+    """
+       third
+    """
+  # Negative margin
+  stdout
+    """
+  forth
```

Comment:
@andreoss I think we should fail in this situation with a "bad syntax" message

Reply:
@yegor256 This would be a rather nontrivial change, the indentation level needs to be stored and compared within quotes. I suggest leaving it to another task.

Reply:
@andreoss sure, let's do it separately, but please add TODO puzzle